notebooks/04_basic_parameters_tuning.ipynb

```diff
@@ -94,14 +94,15 @@
     "    'workclass', 'education', 'marital-status', 'occupation',\n",
```

Comment:
<p>We might have an issue with the path <code>../figures/...</code>  in Windows. It might be better to use <code>os.path.join</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lesteve/scikit-learn-tutorial/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>I am pretty sure that it works fine even on Windows.</p>